Added requirements.txt & updated README (#5)
XanaduAI/quantum-transfer-learning: README.md

The notebooks `q2c_transfer_learning.ipynb` and  `q2q_transfer_learning.ipynb` require the library Strawberry Fields with the TensorFlow backend.

> All notebooks require the package versions specified in the `requirements.txt` to run on a CUDA device.

Reviewer: ```suggestion
Notebooks can be run using the package versions specified in the `requirements.txt` file.
```
Perhaps this way could present a nice "fall-back" for anyone experiencing compatibility issues, regardless of using CUDA or not.
